lib/go-tc/deliveryservice_servers.go

```diff
@@ -168,6 +168,44 @@ type DSServerBaseV4 struct {
 	DeliveryServiceCapabilities []string             `json:"-" db:"deliveryservice_capabilities"`
 }
 
+// DSServerBaseV5 contains the base information for a Delivery Service Server associated with APIv5.
+type DSServerBaseV5 struct {
```

Comment:
The reason we have a DSServerBase was to re-use a struct that shares many details between v4 and earlier versions. You are only using DSServerBaseV5 in exactly one place: the definition of DSServerV50. So it's completely unnecessary.